src/models/collaboration/model.js

```diff
@@ -0,0 +1,86 @@
+export const states = {
```

Comment:
seems like the name of this file should be `utils.js`. otherwise this is really nice

Reply:
moved to `collaborationUtils.js` in https://github.com/WildMeOrg/codex-frontend/pull/451/commits/62104e7a6d783092e520aba2ad9bd9b3ee349b91.